### PR TITLE
s/overriden/overridden

### PR DESCRIPTION
**Pull request by PhilETaylor**
s/overriden/overridden

### CODE REVIEW

File: build/media/vendor/jquery-ui/js/jquery.ui.core.js [hunk 1]
```diff
@@ -1015,7 +1015,7 @@ $.widget("ui.mouse", {
 		return this.mouseDelayMet;
 	},
 
-	// These are placeholder methods, to be overriden by extending plugin
+	// These are placeholder methods, to be overridden by extending plugin
```
Comment (SharkyKZ) [inline build/media/vendor/jquery-ui/js/jquery.ui.core.js]: Revert this please. It's a 3rd party file.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on d8831f0fae1263c88ea4e063c8ece7d86a677131<br /><br />Code review + check that this PR covers all occurrences of wrong English "overriden" (case-insensitive) in Joomla 4.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22998">issues.joomla.org/tracker/joomla-cms/22998</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on d8831f0fae1263c88ea4e063c8ece7d86a677131<br /><br />blind search and replace is dangerous. please dont change third party files. Any issues with them should be reported upstream<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22998">issues.joomla.org/tracker/joomla-cms/22998</a>.</sub>

**Comment by richard67** [comment]
I have not tested this item.<br /><br />Reverting test for reasons stated above by Brian in his negative test result and .<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22998">issues.joomla.org/tracker/joomla-cms/22998</a>.</sub>

**Comment by PhilETaylor** [comment]
One file reported upstream here https://github.com/jquery/jquery-ui/pull/1876

File: build/media/vendor/jquery-ui/js/jquery.ui.core.js (hunk 1 above)
Comment (PhilETaylor) [reply build/media/vendor/jquery-ui/js/jquery.ui.core.js]: Done.

**Comment by PhilETaylor** [comment]
After some struggles with Github Online Editor ALWAYS adding a blank line at the end of the file, the vendor file is now reverted, and reported upstream, this is now ready for "testing" again.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on baa1b4f7ed51c0f28e072fc3f32263ccddd8527f<br /><br />Code review + check that this PR covers all occurrences of wrong English "overriden" (case-insensitive) in Joomla 4 except of the 3rd party library file.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22998">issues.joomla.org/tracker/joomla-cms/22998</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on baa1b4f7ed51c0f28e072fc3f32263ccddd8527f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22998">issues.joomla.org/tracker/joomla-cms/22998</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22998">issues.joomla.org/tracker/joomla-cms/22998</a>.</sub>

**Pull request merged**